gst/nnstreamer/tensor_query/tensor_query_filter.c

```diff
@@ -0,0 +1,121 @@
+/* SPDX-License-Identifier: LGPL-2.1-only */
+/**
+ * Copyright (C) 2021 Samsung Electronics Co., Ltd.
+ *
+ * @file    tensor_query_filter.c
+ * @date    09 Jul 2021
+ * @brief   GStreamer plugin to handle tensor query filter
+ * @author  Junhwan Kim <jejudo.kim@samsung.com>
+ * @see     http://github.com/nnstreamer/nnstreamer
+ * @bug     No known bugs
+ *
+ */
+#ifdef HAVE_CONFIG_H
+#include <config.h>
+#endif
+
+#include "tensor_query_filter.h"
+
+GST_DEBUG_CATEGORY_STATIC (gst_tensor_query_filter_debug);
+#define GST_CAT_DEFAULT gst_tensor_query_filter_debug
+
+/**
+ * @brief the capabilities of the inputs.
+ */
+static GstStaticPadTemplate sinktemplate = GST_STATIC_PAD_TEMPLATE ("sink",
+    GST_PAD_SINK,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS (GST_TENSORS_FLEX_CAP_DEFAULT));
+
+/**
+ * @brief the capabilities of the outputs.
+ */
+static GstStaticPadTemplate srctemplate = GST_STATIC_PAD_TEMPLATE ("src",
+    GST_PAD_SRC,
+    GST_PAD_ALWAYS,
+    GST_STATIC_CAPS (GST_TENSORS_FLEX_CAP_DEFAULT));
+
+#define gst_tensor_query_filter_parent_class parent_class
+G_DEFINE_TYPE (GstTensorQueryFilter, gst_tensor_query_filter,
+    GST_TYPE_BASE_TRANSFORM);
+
+/* GObject vmethod implementations */
+static void gst_tensor_query_filter_set_property (GObject * object,
+    guint prop_id, const GValue * value, GParamSpec * pspec);
+static void gst_tensor_query_filter_get_property (GObject * object,
+    guint prop_id, GValue * value, GParamSpec * pspec);
+static void gst_tensor_query_filter_finalize (GObject * object);
+
+/**
+ * @brief initialize the class
+ */
+static void
+gst_tensor_query_filter_class_init (GstTensorQueryFilterClass * klass)
+{
+  GObjectClass *gobject_class;
+  GstElementClass *gstelement_class;
+  // GstBaseTransformClass *trans_class;
+
+  gobject_class = (GObjectClass *) klass;
+  gstelement_class = (GstElementClass *) klass;
+  // trans_class = (GstBaseTransformClass *) klass;
```

Comment:
There's the same mistake here and elsewhere.

```suggestion
  trans_class = (GstBaseTransformClass *) klass;
  gstelement_class = (GstElementClass *) trans_class;
  gobject_class = (GObjectClass *) gstelement_class;
```

Reply:
Updated!